homeassistant/components/home_connect/translations/nl.json

```diff
@@ -1,7 +1,8 @@
 {
     "config": {
         "abort": {
-            "missing_configuration": "Het component is niet geconfigureerd. Volg de documentatie."
+            "missing_configuration": "Het component is niet geconfigureerd. Volg de documentatie.",
+            "no_url_available": "Geen URL beschikbaar. Voor meer informatie over de fout, [raadpleeg de help sectie]({docs_url})"
```

Comment:
Please don't change translation files besides the default English file.

Reply:
Reverted change on translation file.